src/hdf5/tropics_2ioda.py

```diff
@@ -134,10 +138,15 @@ def get_data_from_files(afile):
     obs_data = init_obs_loc()
 
     f = h5py.File(afile, 'r')
-    software_version = f.attrs['L1b_SW_Ver'].decode("utf-8").split('.')
-    if int(software_version[0]) >= 3:
+    software_version = True
+    if 'L1b_SW_Ver' in f.attrs.keys():
+        software_version = int(f.attrs['L1b_SW_Ver'].decode("utf-8").split('.')[0]) >= 3
+    nc_properties = True
+    if '_NCProperties' in f.attrs.keys():
+        nc_properties = 'version=2' in f.attrs['_NCProperties'].decode("utf-8")
+    if software_version or nc_properties:
         obs_data = get_data(f, obs_data)
-    elif int(software_version[0]) == 2:
+    elif int(f.attrs['L1b_SW_Ver'].decode("utf-8").split('.')[0]) == 2:
```

Comment:
no update from tomorrow on this, but will use this until they reprocess and see if this attribute gets added, will keep asking

Reply:
It's good to have it in place already.